packages/calcite-components/src/components/tabs/tabs.tsx

```diff
@@ -25,15 +26,21 @@ export class Tabs {
   @Prop({ reflect: true }) layout: TabLayout = "inline";
 
   /**
-   * Specifies the position of the component in relation to the `calcite-tab`s.
+   * Specifies the position of `calcite-tab-nav` and `calcite-tab-title` components in relation to the `calcite-tabs`, defaults to `top`.
    */
   @Prop({ reflect: true }) position: TabPosition = "top";
 
   /**
-   * Specifies the size of the component.
+   * Specifies the size of the component, defaults to `m`.
    */
   @Prop({ reflect: true }) scale: Scale = "m";
 
+  @Watch("position")
+  @Watch("scale")
+  handlePropsChange(): void {
```

Comment:
`handlePropsChange` is a bit vague. Can you rename it to better convey what it does? It could be something simple as `handlePositionScaleChange` or `handleInheritableProps`